.github/workflows/unitctl.yml

```diff
@@ -0,0 +1,132 @@
+
+name: unitctl
+
+on:
+  pull_request:
+    paths:
+      - tools/unitctl/**
```

Comment:
Does this actually need to run on pull requests?

For releases it will surely only be once a unit release tag is pushed to the repository?

Reply:
I don't why you wouldn't want it to? It won't kick off the release but it will attempt to compile `unitctl`.

If / when there are tests, they can also be added as a step that runs on PRs.

Reply:
Fair enough.